go2_simulation/bullet_wrapper.py

```diff
@@ -1,27 +1,23 @@
 import numpy as np
-import example_robot_data as erd
+from go2_description import GO2_DESCRIPTION_URDF_PATH
 import os
 import pybullet
 import pybullet_data
 from scipy.spatial.transform import Rotation as R
 
 class BulletWrapper():
     def __init__(self, timestep):
-        ########################## Load robot model and geometry
-        robot_subpath = "go2_description/urdf/go2.urdf"
-        self.robot_path = os.path.join(erd.getModelPath(robot_subpath), robot_subpath)
-        self.robot = 0
         self.init_pybullet(timestep)
 
-    def init_pybullet(self, timestep):
+    def init_pybullet(self, urdf_path, timestep):
```

Comment:
Why add urdf_path_ as argument to init_pybullet?

Reply:
oops my bad, I forgot to remove it, thx